src/ddmd/func.d

```diff
@@ -144,14 +144,16 @@ public:
     }
 }
 
-enum FUNCFLAGpurityInprocess  = 1;      /// working on determining purity
-enum FUNCFLAGsafetyInprocess  = 2;      /// working on determining safety
-enum FUNCFLAGnothrowInprocess = 4;      /// working on determining nothrow
-enum FUNCFLAGnogcInprocess    = 8;      /// working on determining @nogc
-enum FUNCFLAGreturnInprocess  = 0x10;   /// working on inferring 'return' for parameters
-enum FUNCFLAGinlineScanned    = 0x20;   /// function has been scanned for inline possibilities
-enum FUNCFLAGinferScope       = 0x40;   /// infer 'scope' for parameters
-
+enum FUNCFLAG : uint
```

Comment:
Since we are converting those to D, maybe the code style should correspond to the D style.